Framework/Subset_of_CorLib/System/DateTime.cs

```diff
@@ -93,13 +88,18 @@ public struct DateTime
         private const ulong TickMask = 0x7FFFFFFFFFFFFFFFL;
         private const ulong UTCMask = 0x8000000000000000L;
 
-        public static readonly DateTime MinValue = new DateTime(MinTicks);
-        public static readonly DateTime MaxValue = new DateTime(MaxTicks);
+        // Ticks at 12:00 AM January 1, year 1601 A.D.

```

Comment:
While definitely easier the amount of risk potentially involved wasn't really expressed, which could be helpful to contributors facing the same question whether to fix in managed or native code. Can we perhaps put some guidance on the wiki?